os_clean_images.sh

```diff
@@ -7,15 +7,15 @@ set -e
 sudo sed -i /etc/hosts -e "s/^127.0.0.1 localhost$/127.0.0.1 localhost $(hostname)/"
 
 # Installing necessary tool for the script: python-glanceclient
-sudo pip install --upgrade python-glanceclient
+sudo pip install python-openstackclient==3.11.0
```

Comment:
Switching to `python-openstackclient==3.11.0` in lieu of using `python-glanceclient`